pkg/logging/api.go

```diff
@@ -0,0 +1,25 @@
+// Package logging API for structured logging
+package logging
+
+// The global logger instance
+var logger Logger //nolint: gochecknoglobals
+
+// InitLogger initialize the global logger instance
+func InitLogger(logFile string) error {
+	logger, err := newLogrusLogger(logFile)
+	if err != nil {
+		initLogger(logger)
+	}
+
+	return err
```

Comment:
Shouldn't it be opposite?

```suggestion
	logger, err := newLogrusLogger(logFile)
	if err != nil {
		return err
	}
	
	initLogger(logger)
	
	return nil
```

Reply:
Yep! 😅

Reply:
Fixed @yngvark